src/components/button/component.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cx from 'classnames';
+
+import styles from './styles.module.scss';
+
+const tooltipStyles = {
+  'data-place': 'right',
+  'data-effect':'solid',
+  'data-delay-show': 0,
+}
+
+const Component = ({
+  type,
+  Icon,
+  label,
+	className,
+  handleClick,
+  tooltipText
+}) => (
+  <button 
+    onClick={handleClick}
+    data-tip={tooltipText}
+    {...tooltipStyles}
+		className={cx(
+      className,
+      {
+        [styles.rounded]: type === 'rounded',
```

Comment:
I like this separation between style/layout 👍   We used 'variant' in Trase. But type also works :) Maybe I would recommend to put the className variable before this if we want to override something in some weird case (we shouldn't as the className should be only used for layout)